### DISCUSSION TITLE
Increment Arrows Number Fields

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#7990] [Ideas]
Hey,

is there a solution to adding the increment arrows into number fields?
They would be cool to enable/disable.

Sincerely,
Kevin

**Comment by melohagan** [comment]
Could do that, or could create a new custom component for number counter form component.

**Comment by Den4ik1992** [comment]
Does anyone already have a solution ?
Actually, yes simple it should look like here:

https://codepen.io/ChynoDeluxe/pen/poyNEay

But I am not a programmer and don't know how to implement it.
Would be cool if someone can build such a plugin.

**Comment by melohagan** [comment]
Hey @kasgithub8 @Den4ik1992 

I've created a basic number field with incrementing arrows here: https://github.com/melohagan/budibase-component-number-counter-input

It's based off this number input: https://www.svelteui.org/core/number-input/

The full set of options haven't been added, but feel free to fork the repo!

**Reply by Den4ik1992** [reply]
Wow this is exactly what I was looking for. I'll add it to the company dashboard right away, thanks!

You are the best !
